src/chrome/content/rules/Something-Awful.xml

```diff
@@ -1,21 +1,29 @@
-<!--
-	Nonfunctional subdomains:
-
-		- (www.)	(shows secure)
-		- fi
-		- forums	(times out)
-		- i
-
--->
 <ruleset name="Something Awful (partial)">
 
+	<target host="somethingawful.com" />
+	<target host="www.somethingawful.com" />
 	<target host="secure.somethingawful.com" />
+	<target host="forums.somethingawful.com" />
+	<target host="archives.somethingawful.com" />
+	<target host="static.somethingawful.com" />
+	<target host="downloads.somethingawful.com" />
+	<target host="i.somethingawful.com" />
+	<target host="images.somethingawful.com" />
+	<target host="fi.somethingawful.com" />
+	<target host="forumimages.somethingawful.com" />
+	<target host="v.somethingawful.com" />
+	<target host="videos.somethingawful.com" />
 
 
 	<securecookie host="^secure\.somethingawful\.com$" name=".+" />
+	<securecookie host="^forums\.somethingawful\.com$" name=".+" />
+	<securecookie host="^archives\.somethingawful\.com$" name=".+" />
+
 
+	<rule from="^http://somethingawful\.com/"
+		to="https://www.somethingawful.com/" />
```

Comment:
This doesn't matter.
^ will be redirected to https by us, the server redirects to unsecured www then and we will redirect to www over https again.